pkg/plugin/plugin.go

```diff
@@ -100,6 +109,42 @@ func (p *Plugin) SetInformation(parsedURL *url.URL) error {
 	return nil
 }
 
+func findPluginByName(name string, plugins []store.Plugin) *store.Plugin {
+	// for each plugin in the plugins list, check if the name matches the name of the plugin
+	for _, plugin := range plugins {
+		if plugin.Name == name {
+			return &plugin
+		}
+	}
+
+	return nil
+}
+
+func (p *Plugin) checkForPluginUpdates() (bool, error) {
+	// finf if there is an element elem in storeitems that has the same name as rootItem.Name()
+	storeItems, err := store.FetchPluginList()
```

Comment:
We already know the items comes from the store because you're calling the store module

Reply:
I renamed it pluginList, WDYT ?